app/scripts/views/settings/change_password.js

```diff
@@ -24,6 +26,10 @@ define(function (require, exports, module) {
     className: 'change-password',
     viewName: 'settings.change-password',
 
+    events: {
+      'blur input[id=new_password]': 'onPasswordBlur'
```

Comment:
Selector can be same as before : `.password:not("#old_password")`, do we want that here as well? Using `id` here would actually be faster than the other selector, so. LMK if this needs to be changed.